.editorconfig

```diff
@@ -11,24 +11,24 @@ charset = utf-8
 trim_trailing_whitespace = true
```

Comment:
Can you please remove this and add it to either your personal gitignore or the one for this repo?